pkg/repro/repro.go

```diff
@@ -351,6 +355,43 @@ func (ctx *context) repro(entries []*prog.LogEntry, crashStart int) (*Result, er
 	return res, nil
 }
 
+func (ctx *context) repro(entries []*prog.LogEntry, crashStart int) (*Result, error) {
```

Comment:
Put this functions before the functions it calls. In Go it is a common practice to put more important stuff closer to the top of the file and less important stuff at the bottom of the file.